### ISSUE TITLE
Match routes that trigger notification using a regex

### ISSUE THREAD

**Issue by gr2m** [octokit/plugin-throttling.js#4]
With the implementation at #2, the following request would match

```js
octokit.request('POST /repos/:owner/:repo/issues', options)
```

But if someone replaces the URL parameters before sending a request, it wouldn’t:

```js
octokit.request('POST /repos/foo/bar/issues', options)
```

This should be an exception so I don’t think it’s a big problem right now, but it would be better if we would match both requests in future

**Comment by SGrondin** [comment]
@gr2m  Is `feat(#4):` the right prefix for this?

**Comment by gr2m** [comment]
I would just use `feat: `, but that works too :)

**Comment by gr2m** [comment]
closed via #11
